src/app/visual/page/index.html

```diff
@@ -0,0 +1,3 @@
+<sky-page>
```

Comment:
I noticed that once you load this visual and navigate away (using the back button), all other pages now have the white background, even if they're not using the `sky-page` component.

Reply:
Looks like this is sorted out now. Thanks!